charts/testkube-cloud-api/values.yaml

```diff
@@ -30,6 +30,8 @@ global:
   certManager:
     # -- Certificate Issuer ref (only used if `provider` is set to `cert-manager`)
     issuerRef: ""
+  # -- Custom CA to use as a trusted CA during TLS connections. Specify a secret with the PEM encoded CA under the ca.crt key.
+  customCaSecretRef: ""
```

Comment:
Duplicate this value to enterprise values.yaml also

Reply:
Done.